src/ark/segmentation/ez_seg/ez_object_segmentation.py

```diff
@@ -154,7 +154,9 @@ def _create_object_mask(
     misc_utils.verify_in_list(object_shape_type=[object_shape_type], object_shape_options=["blob", "projection"])
 
     # Copy the input image, and get its shape
-    img2mask: np.ndarray = input_image.copy().to_numpy()
+    img2mask: np.ndarray = input_image.copy()
```

Comment:
Where I'm implementing it in masking_utils, `img2mask` is already an np.array, so I get an AttributeError: 'numpy.ndarray' object has no attribute 'to_numpy'.

Reply:
The ezseg functions only save the composites, they don't return them so I would have to change the ezseg notebook to handle a return (not a big deal, but unnecessary when it's not even used). Since the only case we would need the composites returned is for the internal function call inside `generate_signal_masks()`, I didn't think it was needed.